### PR TITLE
[DDW-940] Use the new faker.js npm module

### PR DESCRIPTION
**Pull request by marcin-mazurek**
This PR replaces the unpublished faker.js library with its successor.

## Todos

- [ ] TODO

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

### Basics

- [x] PR assigned to the PR author(s)
- [x] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [ ] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [ ] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [ ] PR link is added to a Jira ticket, ticket moved to In Review
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR contains screenshots (in case of UI changes)
- [ ] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by danielmain** [comment]
Stupid question @marcin-mazurek what was wrong with the old one 🤔?

**Comment by marcin-mazurek** [comment]
@danielmain In the Slack thread Tomislav raised a concern about potential impact on installers due to changing externals, however I checked installers on all platforms and they are not affected, Daedalus successfully installs and launches. I believe we should be able to merge this one

File: source/renderer/app/config/generateStakePoolFakeData.ts [hunk 1]
```diff
@@ -1,33 +1,31 @@
 /**
  * It generates stake pool dummy json content
- * Command to run: node source/renderer/app/config/generateStakePoolFakeData.js
+ * Command to run: npx ts-node source/renderer/app/config/generateStakePoolFakeData.ts
```
Comment (szymonmaslowski) [inline source/renderer/app/config/generateStakePoolFakeData.ts:3]: As we use `yarn` we should also suggest to use it to run this script.
```suggestion
 * Command to run: yarn ts-node source/renderer/app/config/generateStakePoolFakeData.ts
```

**Review by szymonmaslowski — APPROVED** [review]
Good job, just one small suggestion.

File: source/renderer/app/config/generateStakePoolFakeData.ts (hunk 1 above)
Comment (marcin-mazurek) [reply source/renderer/app/config/generateStakePoolFakeData.ts:3]: good catch

**Pull request merged**